docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+# OpenShift + Maven Tutorial
+
+While developing your application with Java, you'll most likely be building
+many, many, many times.  In order to shorten the build times of your pods,
+dependencies can be cached in local Maven repositories.  This tutorial will step
```

Comment:
Yes indeed, I may have used the wrong terms.  Wouldn't the ~/.m2/repository directory die when the builder pod is destroyed?